test/routes/api-documentation.test.ts

```diff
@@ -0,0 +1,40 @@
+import jsonld from 'jsonld';
+import { Next, Response } from 'koa';
+import apiDocumentation from '../../src/routes/api-documentation';
+import runMiddleware from '../middleware';
+import createContext from './context';
+
+const makeRequest = async (next?: Next): Promise<Response> => {
+  const context = createContext();
+
+  return runMiddleware(apiDocumentation(context.router), context, next);
+};
+
+describe('API documentation', (): void => {
+  it('should return a successful response', async (): Promise<void> => {
+    const response = await makeRequest();
+
+    expect(response.status).toBe(200);
+    expect(response.type).toBe('application/ld+json');
+  });
+
+  it('should return the API documentation', async (): Promise<void> => {
+    const response = await makeRequest();
+    const graph = await jsonld.expand(response.body);
+
+    expect(graph).toHaveLength(1);
+
+    const object = graph[0];
+
+    expect(object['@id']).toBe('http://example.com/path-to/api-documentation');
+    expect(object['@type']).toContain('http://www.w3.org/ns/hydra/core#ApiDocumentation');
```

Comment:
meaning the `@type` is an array after expansion?

Reply:
Yes, can be more than one.